packages/ic-response-verification/src/types/verification_result.rs

```diff
@@ -1,114 +1,62 @@
-use crate::{types::VerifiedResponse, ResponseVerificationError};
+use crate::types::VerifiedResponse;
 
 #[cfg(all(target_arch = "wasm32", feature = "js"))]
 use wasm_bindgen::prelude::*;
 
-#[cfg(all(target_arch = "wasm32", feature = "js"))]
-use crate::ResponseVerificationJsError;
-
 #[cfg(all(target_arch = "wasm32", feature = "js"))]
 #[wasm_bindgen(typescript_custom_section)]
 const VERIFICATION_RESULT: &'static str = r#"
-type VerificationResult = {
-  passed: true;
+type VerificationInfo = {
   response?: VerifiedResponse;
```

Comment:
Since we don't have `passed` anymore is there any situation where the response would not be set?

Reply:
Yes, you can still skip certification for a given path entirely with a CEL expression. In this case we return `None` for the `response`: https://github.com/dfinity/response-verification/blob/main/packages/ic-response-verification/src/verification/verify_request_response_pair.rs#L253-L261

Reply:
I think we are currently not handling that scenario then from the clients of response verification, (e.g. [service-worker returns a failed verification response](https://github.com/dfinity/ic/blob/205fc8bb8c82238f0a750e7e0464da864c8b3d39/typescript/service-worker/src/sw/requests/query-call.ts#L40))